<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="CAVICSRGXFTVT2" name="CTD.Dynamic_Adapter_EndpointsTest" tests="7" failures="1" timestamp="19 May 2019 03:02:24 GMT" time="0.986" errors="0">
  <testcase name="disciplinarychargeregulation" time="0.048" classname="CTD.Dynamic_Adapter_EndpointsTest"/>
  <testcase name="transferreason" time="0.039" classname="CTD.Dynamic_Adapter_EndpointsTest"/>
  <testcase name="disciplinarycharge" time="0.069" classname="CTD.Dynamic_Adapter_EndpointsTest"/>
  <testcase name="cliententity" time="0.600" classname="CTD.Dynamic_Adapter_EndpointsTest"/>
  <testcase name="tombstoneadult" time="0.095" classname="CTD.Dynamic_Adapter_EndpointsTest"/>
  <testcase name="transfer" time="0.041" classname="CTD.Dynamic_Adapter_EndpointsTest">
    <failure type="java.lang.AssertionError" message="clientdatabind element validation expected [/contacts(ssg_csnumber=04704102)] but found [84]">
      <![CDATA[java.lang.AssertionError: clientdatabind element validation expected [/contacts(ssg_csnumber=04704102)] but found [84]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at CTD.Dynamic_Adapter_EndpointsTest.transfer(Dynamic_Adapter_EndpointsTest.java:379)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- transfer -->
  <testcase name="tombstoneyouth" time="0.094" classname="CTD.Dynamic_Adapter_EndpointsTest"/>
</testsuite> <!-- CTD.Dynamic_Adapter_EndpointsTest -->
